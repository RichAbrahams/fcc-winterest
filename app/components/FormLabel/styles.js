import styled from 'styled-components';

export const Label = styled.label`
  margin: 1em 0em 0.5em 0em;
`;
