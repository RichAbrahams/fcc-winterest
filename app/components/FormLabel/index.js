/**
*
* FormLabel
*
*/

import React from 'react';
import { Label } from './styles';


function FormLabel(props) {
  return (
    <Label>
      {props.children}
    </Label>
  );
}

FormLabel.propTypes = {
  children: React.PropTypes.string,
};

export default FormLabel;
