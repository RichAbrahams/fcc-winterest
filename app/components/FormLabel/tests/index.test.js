import React from 'react';
import renderer from 'react-test-renderer';
import FormLabel from '../index';

it('renders correctly', () => {
  const tree = renderer.create(
    <FormLabel />).toJSON();
  expect(tree).toMatchSnapshot();
});
