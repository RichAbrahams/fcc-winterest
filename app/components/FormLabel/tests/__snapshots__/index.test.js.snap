exports[`test renders correctly 1`] = `
<label
  className="bgKYfS" />
`;
